### ISSUE TITLE
Suggest

### ISSUE THREAD

**Issue by Cromo74** [bangstk/MPK2_Plus_for_Ableton#2]
Hi, I’m an end user of Akai MPK261 and Ableton Live 11 Suite 10.0.2 installed on a IMac MacOs High Sierra version 10.13.6
I’m very interested on features you implemented in your script. I’ve already tried other scripts and they works but with some lacks (scripts kindly edited by users Steve-gh and Max Cloutier arevery useful). A little question for a cosmetic issue: is it necessary to install all three folders to see script in control surface. I tried  to put only folder MPK2_Plus_MPK261 in Application/Ableton Live/Contents/App-Resources/MIDI Remote Scrips, but I can’t find it in the control surface list under MIDI Preferences. I have to put all three folder also if I need just MPK261 script?  What I can suggest you is to assign to other PAD banks drum functionality of course but also other useful button like TAP tempo, metronome, and the other buttons included in transport control (. Another bank can be dedicated to scene management.Anyway, thanks for your work! Great job! Claudio

**Comment by Cromo74** [comment]
Here there are some features suggestions...https://remotify.io/product/purchase...
Last question...is it normal that MIDI Out indicator and MIDI track in indicator are permanently blinking? Is it due to red box?
Thanks
Claudio

**Comment by bangstk** [comment]
Yes you need all three folders because the MPK261 script inherits the MPK249 script with slight changes. Maybe I can move the base implementation into the MPK2 Core script folder (which does not show up in the list) but I don't really see much purpose. You select the script once and then you don't need to look at the list again.

Yes MIDI Out will flash because we have to manually update the pad color to do stuff like pulsing or blinking, since it isn't implemented in the hardware like it is on most other newer MIDI keyboards. If this is too big of a deal maybe I can make a variant or option to only use static colors for clips being queued or currently playing/recording.

**Comment by Cromo74** [comment]
Hi,
It’s not a problem at all, it was just a curiosity.
I look forward to the next updates especially for the drum functionality in bank D.
Thanks again for your work.
Claudio

claudiosica.net <http://www.claudiosica.net/>
